pkgs/development/embedded/stm32/stm32cubemx/default.nix

```diff
@@ -52,6 +54,19 @@ stdenv.mkDerivation rec {
           $out/share/icons/hicolor/"$size"x"$size"/apps/${pname}.png
       fi
     done;
+
+    cat << EOF > $out/share/applications/stm32cubemx.desktop
```

Comment:
I would rather see these contents live in a separate `stm32cubemx.desktop` file, which you can then link via string interpolation.

Reply:
Used `makeDesktopItem` instead.